presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -32,6 +34,15 @@
     private boolean caseInsensitiveNameMatching;
     private Duration caseInsensitiveNameMatchingCacheTtl = new Duration(1, MINUTES);
     private Set<String> jdbcTypesMappedToVarchar = ImmutableSet.of();
+    private Integer decimalDefaultScale = 16;
+    private DecimalOverMaxPrecisionMapping decimalOverMaxPrecisionMapping = DecimalOverMaxPrecisionMapping.SKIP;
+
+    public enum DecimalOverMaxPrecisionMapping {
```

Comment:
Use java.math.RoundingMode instead.

Reply:
`java.math.RoundingMode` is not sufficient. We need also `SKIP`

Reply:
@kokosing It's possible to use `java.math.RoundingMode` enum if we give up the option to preserve the current behaviour of skipping (hiding) such columns.